modules/GenericTools/src/GenericTools/Model/baseTable.php

```diff
@@ -230,7 +230,9 @@ public function buildGenericSelect(array $FilterData = null,$order = null,$speci
 
         $where = new Where();
         foreach($FilterData as $field=>$value){
-            $last = self::$AND;
+
+
+            $last = (is_null($value[0]['sqlOp'])) ? self::$AND : $value[0]['sqlOp'];
```

Comment:
need to test that  this change works well with requests with additional parameters

Reply:
LIKE ? AND is default  and there is only OR .. What other param can be ?